frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx

```diff
@@ -24,12 +28,13 @@ const filtersToLocalFilters = (filters: RecordingFilters): LocalRecordingFilters
 
     return {
         actions: [],
-        events: [
+        events: [],
+        new_entity: [
             {
-                id: '$pageview',
+                id: 'empty',
```

Comment:
I left $pageview as the default because all events was super slow 🙈

Reply:
(non-blocking here, I'd leave it as pageview so it feels snappier and see how many people change it, but no harm in doing the reverse)

Reply:
In this case now it is simply "not there" which is I think preferable but as you say - lets see what people do